docs/development/Building in Windows 10 or 11 with MSYS2.md

```diff
@@ -54,21 +61,21 @@ This will give you the version you need for any given release or master branch.
 
 https://github.com/xpack-dev-tools/arm-none-eabi-gcc-xpack/releases/
 ```
-# for version 2.6.1, version needed is 9.2.1
-wget https://github.com/xpack-dev-tools/arm-none-eabi-gcc-xpack/releases/download/v9.2.1-1.1/xpack-arm-none-eabi-gcc-9.2.1-1.1-win32-x64.zip
-unzip xpack-arm-none-eabi-gcc-9.2.1-1.1-win32-x64.zip
+# for iNav version 5.0.0, tookchain version needed is 10.2.1
```

Comment:
Let's use `INAV` everywhere. All capitals please :)

Reply:
Exactly!
And please correct "tookchain"

Reply:
Done!